units/vi/unit1/what-are-llms.mdx

```diff
@@ -150,7 +150,7 @@ Bạn có thể tương tác với quá trình decoding của SmolLM2 trong Spac
 	height="450"
 ></iframe>
 
-- Có những chiến lược nâng cao hơn như *beam search*: khám phá nhiều chuỗi ứng viên để tìm chuỗi có tổng điểm cao nhất.
+- Có những chiến lược nâng cao hơn như *tìm kiếm chùm (beam search)*: khám phá nhiều chuỗi ứng viên để tìm chuỗi có tổng điểm cao nhất.
```

Comment:
Nice, I didn't know about the word "tìm kiếm chùm" before, now I know it's a valid term in vi 👍